framework/source/class/qx/bom/client/Html.js

```diff
@@ -353,7 +353,7 @@ qx.Bootstrap.define("qx.bom.client.Html",
      * @ignore(Image)
      */
     getDataUrl : function(callback) {
-      var data = new Image();
+      var data = new window.Image();
```

Comment:
it suppresses the warnings because `window` is always an allowed/well known global but `Image` isn't when compiling for a NodeJS target.  Part of my goal was to improve NodeJS development, by adding VDOM and reducing the unnecessary dependencies (because this can cause code to be loaded which crashes) - connected with this was removing warnings which are issued on nodejs-only targets.

In any code which has to be cross platform, rather than adding a blanket `@ignore(window.*)` instead I tend to use `window.Image` because it's direct about what the code is referencing